docs/translations/lazy-load-translations.md

```diff
@@ -0,0 +1,138 @@
+# Lazy-load translations
+
+Some of our translations, are lazy-loaded. To lazy-load them, the translation chunks needs to be registered. This takes place in `/src/BlockTypes/AbstractBlock.php`:
```

Comment:
> Also, I think we should explain why do we need to lazy-load translation.

From what I understood, we don't need to lazy-load them, but decided doing so, so that only relevant translations are loaded on the corresponding pages. So, instead of loading all translations when a shopper enters a shop, the translations of the Cart page or the Checkout page are only loaded once the shopper enters these pages.

Have I misunderstood that and may I ask what you had in mind, why we need to lazy-load translations, @dinhtungdu?

Reply:
I've adjusted the section with information about lazy-loading. Would you mind doing another review, @dinhtungdu?